### PR TITLE
Migrated all tests to phpunit 6.x

### PR DESCRIPTION
**Pull request by TomHAnderson**
A complete tidy-ing up of this repository with correct phpcs, phpunit, and php version support.  This should be applied before #31 or #32

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,26 +24,27 @@ env:
...
+        - EXECUTE_CS_CHECK=true
     - php: 7
       env:
         - SERVICE_MANAGER_VERSION="^2.7.5"
-  allow_failures:
-    - php: hhvm
+    - php: 7.1
+      env:
+        - EXECUTE_CS_CHECK=true
+    - php: 7.1
```
Comment (ezimuel) [inline .travis.yml]: We should add also PHP 7.2

File: composer.json [hunk 2]
```diff
@@ -14,12 +14,13 @@
...
     "require": {
-        "php": "^5.5 || ^7.0"
+        "php": "^5.6 || ^7.0"
     },
     "require-dev": {
         "phpunit/PHPUnit": "~5.0 || ~6.0",
         "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3",
-        "zendframework/zend-coding-standard": "^1.0"
+        "zendframework/zend-coding-standard": "^1.0",
+        "doctrine/instantiator": "~1.0 || ^1.5"
```
Comment (ezimuel) [inline composer.json]: Why do we need `doctrine/instantiatior`?

**Review by ezimuel — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (TomHAnderson) [reply composer.json]: I'll remove this.  It was a dev issue I thought was an issue.

**Review by ezimuel — APPROVED** [review]

**Comment by ezimuel** [comment]
@TomHAnderson Thanks for your contribution. Now, I'm going to review #31 and #32.

**Pull request merged**